nix/tests/http-mock-server.py

```
@@ -237,10 +237,14 @@ def run_server(port=None):
         port_file = os.environ.get("HTTP_MOCK_PORT_FILE", "/tmp/http-mock-port")
         try:
+            # Ensure directory exists
+            os.makedirs(os.path.dirname(port_file), exist_ok=True)
             with open(port_file, "w") as f:
                 f.write(str(port))
-        except:
-            pass  # Ignore if we can't write the port file
+            print(f"Port file written to {port_file}")
+        except Exception as e:
+            print(f"ERROR: Failed to write port file {port_file}: {e}")
+            raise e  # Don't ignore port file write errors
```

Ruff (0.14.14)

[warning] Use `raise` without specifying exception name

Remove exception name

(TRY201)